cms/djangoapps/contentstore/tests/test_proctoring.py

```diff
@@ -74,6 +75,10 @@ def test_publishing_exam(self, is_time_limited, default_time_limit_minutes,
         a proctored exam, it will also put an entry into the exam tables
         """
 
+        hide_after_due = False
+        if not is_proctored_exam and not is_practice_exam:
```

Comment:
Is it possible to just always allow "hide_after_due"? That would simplify the code, and I don't see a compelling reason to disallow it in a couple of cases.